src/lib/tls/tls_messages.h

```diff
@@ -20,17 +20,17 @@
 #include <string>
 #include <set>
 
-namespace Botan {
-
-class Credentials_Manager;
+#if defined(BOTAN_HAS_CECPQ1)
+  #include <botan/cecpq1.h>
+#endif
 
```

Comment:
Not ideal, but acceptable as long as the included headers are all public (as here).